tutorials/create-cloud-build-image-factory-using-packer/index.md

```diff
@@ -41,59 +59,297 @@ gcloud config set compute/zone $ZONE
 gcloud config set account $ACCOUNT
 ```
 
-## Set the project variable
+</details>
+
+## Building a Windows Image
+
+<details>
+<summary>Click to reveal commands</summary>
+
+```powershell
+$env:PROJECT="NEW PROJECT NAME"
```

Comment:
project ID, not name

Also, we usually tell people to use Cloud Shell for tutorials because that simplifies a few things: 

* You can assume that gcloud is installed and current. 
* You can use `GOOGLE_CLOUD_PROJECT` to access the current project ID

Reply:
It seems that the project ID suggestion was taken.

What about the suggestion to use Cloud Shell?

Reply:
Updated instructions to suggest Cloud Shell as alternative method

`Alternatively, you can use [Cloud Shell](https://cloud.google.com/shell) from the GCP Console if you don't have gcloud installed on your local machine.`